csvcubed/csvcubed/readers/preconfiguredtemplates.py

```diff
@@ -0,0 +1,78 @@
+"""
+Pre-configured JSON Templates
+-----------------------------
+
+Functionality to help augment JSON files with configuration from some pre-configured templates.
+"""
+import logging
+
+from typing import Dict, Any
+from requests.exceptions import JSONDecodeError
+
+from csvcubed.utils.cache import session
+
+def get_template_file_from_template_lookup(template_value: str, version_module_path: str) -> str:
+    """
+    Given the `from_template` value, look up the template in the git repo
+    """
+
+    template_lookup_url =  f"https://raw.githubusercontent.com/GSS-Cogs/csvcubed/main/csvcubed/csvcubed/readers/{version_module_path}/templates/preset_column_config.json"
+    template_lookup_response = session.get(template_lookup_url)
+    logging.debug("The template is at: %s", template_lookup_url)
+   
+    if template_lookup_response.status_code != 200:
+        logging.warning("Status code: %s.", template_lookup_response.status_code)
+
+    try:
+        template_lookup = template_lookup_response.json()
+    except JSONDecodeError as e:
+        raise Exception(f"Could not access template lookup file at {template_lookup_url}") from e
+
+    template_file = template_lookup.get(template_value)
+
+    return template_file
```

Comment:
Technicaly the return type here should be `Optional[str]` since `dict.get(...)` will return `None` if it can't find the key.

Perhaps raise an exception if the `template_file` (path) is `None` and let the user know that we couldn't find the template they were looking for?